frontend/components/utils/AnimatedNumber.vue

```diff
@@ -26,10 +31,22 @@ export default Vue.extend({
         },
         decimalPlaces: {
             type: Number,
-            default: 2,
+            default: DECIMAL_PLACES_DEFAULT,
         },
     },
     computed: {
+        displayFullNumber(): boolean {
+            return this.decimals <= DECIMAL_PLACES_MAX || this.value < 0;
+        },
```

Comment:
This means, if the `value` is less than 0, let's say `-0.1` we will `displayFullNumber`, which will, according to the template, display `0.00001`